### ISSUE TITLE
Add option to disable coloring text in damage string by damage type

### ISSUE THREAD

**Issue by shandrakor** [foundryvtt/pf2e#6350]
I came across this post on the reddit. I like the colors myself, but it's possible there's an accessibility issue here?

https://www.reddit.com/r/FoundryVTT/comments/1119476/disable_color_for_damage/

**Comment by CarlosFdez** [comment]
I suspect the actual problem might be that we may have poor color contrast, and maybe some poor color choices. I feel like what might be best is getting specific examples of colors that are problematic, and maybe tweaking the background opacity.

This feels like a solution in search of a question.

**Comment by shandrakor** [comment]
I mean, it's demonstrably a problem because somebody has said they're having a problem. I couldn't say with confidence that removing color entirely is the best solution to that problem, but it's quite clearly A solution to A problem.

**Comment by stwlam** [comment]
Do you have any details of which colors they were having a problem with?

**Comment by shandrakor** [comment]
Nope, I was just passing on a concern that I saw posted on the reddit.

**Comment by stwlam** [comment]
I don't think we should move on this one without more information.
